packages/genesys-spark-components/src/components/stable/gux-dropdown-multi/gux-dropdown-multi.tsx

```diff
@@ -175,7 +175,7 @@ export class GuxDropdownMulti {
    */
   @Method()
   getSelectedValues(): Promise<string[]> {
-    return Promise.resolve(this.value?.split(',') || []);
+    return Promise.resolve(this.value?.split(',').filter(i => i !== '') || []);
   }
 
```

Comment:
I think you could move this logic of converting the value string into an array into one function and then call it in the three locations.

```
private convertValueToArray(value: string): string[] {
  if (value) {
    return value.split(',');
  }

  return [];
}

Reply:
Now is where I find out that something like `value?.<function>` is not quite the same as an if. Empty string returns false in an if statement and tertiary, but returns true for that 🤦‍♂️